services/logger.service.js

```diff
@@ -0,0 +1,49 @@
+const makeService = ({ winston }) => {
+    const logger = winston.createLogger({
+        format: winston.format.simple(),
+    });
+
+    const loggerFormat = winston.format.printf(({ message, timestamp }) => `${timestamp} ${message}`);
+
+    // add output also to console on development
```

Comment:
u cld outsource the things that are in the if and else into funcs then u dont need a comment to explain - ur funcs in if and else wld explain what they do :)